src/main/java/ca/uhn/fhir/jpa/starter/BaseJpaRestfulServer.java

```diff
@@ -351,28 +354,24 @@ protected void initialize() throws ServletException {
     if (appProperties.getImplementationGuides() != null) {
       Map<String, AppProperties.ImplementationGuide> guides = appProperties.getImplementationGuides();
       for (Map.Entry<String, AppProperties.ImplementationGuide> guide : guides.entrySet()) {
-        packageInstallerSvc.install(new PackageInstallationSpec()
-          .setPackageUrl(guide.getValue().getUrl())
-          .setName(guide.getValue().getName())
-          .setVersion(guide.getValue().getVersion())
-          .setInstallMode(PackageInstallationSpec.InstallModeEnum.STORE_AND_INSTALL));
+			PackageInstallOutcomeJson outcome = packageInstallerSvc.install(new PackageInstallationSpec()
```

Comment:
Doesn't seem like this variable assignment is necessary, though I may just be missing its usage.

Reply:
Was used in a debug session. It shouldn't have survived